ToDoList/ToDoList/Modules/TaskDisplay/Core/Model/TaskItem.swift

```diff
@@ -7,21 +7,32 @@
 import UIKit
 
 enum TaskStatus: String, CaseIterable {
-    case active = "Active"
-    case completed = "Completed"
+    case active
+    case completed
+    
+    func initializeLocalization() -> String {
+        switch self {
+        case .active:
+            return TaskDisplayLocalization.tableViewActiveSectionTitle
+        case .completed:
+            return TaskDisplayLocalization.tableViewCompletedSectionTitle
+        }
+    }
 }
 
 struct TaskItem: Hashable {
     var status: TaskStatus
     var title: String
     var description: String?
+    var timeSinceLastChange: Date
     
     mutating func changeTaskDetails(title: String, description: String?) {
         self.title = title
         self.description = description
     }
     
-    mutating func changeTaskStatus(with status: TaskStatus) {
+    mutating func changeTaskStatus(with status: TaskStatus, time timeSinceLastChange: Date) {
         self.status = status
+        self.timeSinceLastChange = timeSinceLastChange
```

Comment:
When changing the status you can just set the timeSinceLastChange to Date() instead of doing it externally. This is model related logic, that can be done inside model
```suggestion
        self.timeSinceLastChange = Date()
```